### PR TITLE
adxl345

### PR DESCRIPTION
**Pull request by Deryugin**
Add driver for adxl345 and simple command for continuously print sensor output

### CODE REVIEW

File: src/cmds/hardware/sensors/adxl345/adxl345.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include <stdint.h>
```
Comment (alexkalmuk) [inline src/cmds/hardware/sensors/adxl345/adxl345.c]: File header?

File: src/cmds/hardware/sensors/adxl345/adxl345.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#include <stdio.h>
+
+#include <drivers/sensors/adxl345.h>
+
+int main(int argc, char **argv) {
+	printf("Testing adxl345 accelerometer output\n");
+
+	adxl345_init();
```
Comment (alexkalmuk) [inline src/cmds/hardware/sensors/adxl345/adxl345.c]: Shouldn't we check for result?

File: src/drivers/sensors/adxl345/adxl345.c [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+#define ADXL345_DATA_AXIS(n)    (ADXL345_DATAX0 + (n))
+
+#define ADXL345_DEVID_VALUE     0xE5
+
+#define ADXL345_BUS    OPTION_GET(NUMBER, bus)
+#define ADXL345_OFFSET OPTION_GET(NUMBER, offset)
+
+struct adxl345_dev {
+	int i2c_bus;
+	int i2c_offset;
```
Comment (alexkalmuk) [inline src/drivers/sensors/adxl345/adxl345.c]: Why do not name this field `i2c_addr`? I think addr is more common for i2s devices than offset.

File: src/cmds/hardware/sensors/adxl345/adxl345.c (hunk 1 above)
Comment (Deryugin) [reply src/cmds/hardware/sensors/adxl345/adxl345.c]: Added

File: src/cmds/hardware/sensors/adxl345/adxl345.c (hunk 2 above)
Comment (Deryugin) [reply src/cmds/hardware/sensors/adxl345/adxl345.c]: Added check for successfull init

File: src/drivers/sensors/adxl345/adxl345.c (hunk 3 above)
Comment (Deryugin) [reply src/drivers/sensors/adxl345/adxl345.c]: You're right. Changed it to `i2c_addr`

**Pull request merged**